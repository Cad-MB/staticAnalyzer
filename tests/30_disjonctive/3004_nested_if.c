{
    int x;
    x = 0;
    if (x < 5) {
      if (x < 2) {
        x = 1;
      } else {
        x = 3;
      }
    } else {
      x = 6;
    }
    print_all;
  }