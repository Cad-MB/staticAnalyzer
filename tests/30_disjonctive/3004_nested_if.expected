tests/30_disjunctive/3004_nested_if.c:11.2-12: [ x in {[1;1] ∨ [3;3] ∨ [6;6]} ]
analysis ended